storybook/pages/AccountContextMenuPage.qml

```diff
@@ -0,0 +1,87 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import AppLayouts.Wallet.views 1.0
+
+import Storybook 1.0
+import Models 1.0
+
+import utils 1.0
+import shared.views.chat 1.0
+
+SplitView {
+    SplitView {
+        orientation: Qt.Vertical
+        SplitView.fillWidth: true
+
+        Rectangle {
+            SplitView.fillWidth: true
+            SplitView.fillHeight: true
+            color: Theme.palette.statusAppLayout.rightPanelBackgroundColor
+            clip: true
+
+            Button {
+                anchors.centerIn: parent
+                text: "Show menu"
+                onClicked: {
+                    menu.createObject(this).popup()
+                }
+            }
+
+            Component {
```

Comment:
There is no need to wrap into `Component` and use `onClosed: destroy()`. Just static component and `onClicked: menu.popup()`.